cmd/sops/common/common.go

```diff
@@ -29,32 +30,37 @@ type ExampleFileEmitter interface {
 	EmitExample() []byte
 }
 
+type Configurable interface {
+	Configure(*config.Config)
+}
```

Comment:
What is this used for? Maybe I missed something, but this never seems to be called, and all implementations don't do anything, not even the one for the YAML store.

Reply:
You are right, it's useless, good catch. It's a remainder of PR #930 from which I started. Maybe he had another idea in mind.